Benchmarks/Schema.NET.Benchmarks/Core/BookBenchmark.cs

```diff
@@ -2,81 +2,79 @@ namespace Schema.NET.Benchmarks.Core
 {
     using System;
     using System.Collections.Generic;
-    using BenchmarkDotNet.Attributes;
 
     public class BookBenchmark : SchemaBenchmarkBase
     {
-        [GlobalSetup]
-        public void Setup() => this.ConfigureBenchmark(new Book()
-        {
-            Id = new Uri("https://example.com/book/1"),
-            Name = "The Catcher in the Rye",
-            Author = new Person()
+        public override Thing InitialiseThing() =>
```

Comment:
Made benchmarks more type safe which also fixes a bunch of null errors. Potentially, this could have been a separate PR.